lib/galaxy/webapps/galaxy/api/datasets.py

```diff
@@ -241,17 +242,30 @@ def update_permissions(
 
     @router.get(
         "/api/histories/{history_id}/contents/{history_content_id}/extra_files",
-        summary="Generate list of extra files.",
+        summary="Get the list of extra files/directories associated with a dataset.",
         tags=["histories"],
+        deprecated=True,
```

Comment:
I wouldn't deprecate this but I suppose it is an unpopular opinion.

Reply:
I thought it was odd, that you need to put a history ID that is not used and will get ignored anyway :sweat_smile:
Also, as far as I know, this applies only to datasets and not to collections.
But I will remove the deprecation flag, I should have included it in the PR description, my bad :sweat_smile: